## :x: Failed noetic 3.20

<details><summary>Log summary</summary>

- [Action Run Summary](https://github.com/openspur/ypspur_ros/actions/runs/20089254207)

## Summary
```
noetic built:	1
noetic tried:	2
noetic deleted:	0
noetic total built:	1
noetic total relevant aports:	2
noetic total aports:	2
```

## ypspur_ros
**ros-noetic-ypspur-ros-0.7.0_git20251210061422-r0.apk**
### Build log
```
CMake Error at /usr/ros/noetic/share/catkin/cmake/python.cmake:4 (find_package):
  By not providing "FindPythonInterp.cmake" in CMAKE_MODULE_PATH this project
  has asked CMake to find a package configuration file provided by
  "PythonInterp", but CMake did not find one.

  Could not find a package configuration file provided by "PythonInterp"
--
-- Configuring incomplete, errors occurred!
<== Failed to process package 'ypspur_ros': 
  Command '['cmake', '/aports/noetic/ypspur_ros/abuild/src/ypspur_ros', '-DCATKIN_DEVEL_PREFIX=/aports/noetic/ypspur_ros/abuild/devel_isolated/ypspur_ros', '-DCMAKE_INSTALL_PREFIX=/aports/noetic/ypspur_ros/abuild/install_isolated', '-DCMAKE_BUILD_TYPE=RelWithDebInfo', '-G', 'Unix Makefiles']' returned non-zero exit status 1.

Reproduce this error by running:
==> cd /aports/noetic/ypspur_ros/abuild/build_isolated/ypspur_ros && cmake /aports/noetic/ypspur_ros/abuild/src/ypspur_ros -DCATKIN_DEVEL_PREFIX=/aports/noetic/ypspur_ros/abuild/devel_isolated/ypspur_ros -DCMAKE_INSTALL_PREFIX=/aports/noetic/ypspur_ros/abuild/install_isolated -DCMAKE_BUILD_TYPE=RelWithDebInfo -G 'Unix Makefiles'

Command failed, exiting.
```

## yp-spur
**ros-noetic-ypspur-1.23.3_git20251125003923-r0.apk**
Build succeeded.
</details>

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 3.0.2...3.28)
+cmake_minimum_required(VERSION 3.0.2...3.26)
 project(ypspur_ros)
 
 find_package(catkin REQUIRED COMPONENTS

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 3.0.2...3.26)
+cmake_minimum_required(VERSION 3.0...3.26)
 project(ypspur_ros)
 
 find_package(catkin REQUIRED COMPONENTS

